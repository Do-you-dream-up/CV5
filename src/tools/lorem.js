/**
 * Export ready-to-use dummy texts.
 */
export const LOREM = {
  links: `
    <h2>Links</h2>
    <p>
      Internal link: <a href>click</a>
      <br />
      External link: <a href target="_blank">click</a>
    </p>
  `,

  lists: `
    <h2>Lists</h2>
    <ol>
      <li>Quis lectus nulla at volutpat diam ut venenatis!</li>
      <li>Nibh nisl.</li>
    </ol>
    <ul>
      <li>Pretium quam vulputate dignissim suspendisse in est ante in nibh mauris.</li>
      <li>Cursus mattis molestie a, iaculis at erat?</li>
    </ul>
  `,

  paragraphs: `
    <h2>Paragraphs</h2>
    <p>Pellentesque massa placerat duis ultricies lacus sed turpis tinciduntid aliquet risus.</p>
    <p>Dictum at tempor commodo.</p>
    <p>Ullamcorper a lacus vestibulum sed? Ullamcorper eget nulla facilisi etiam!</p>
  `,
};

export const LOREM_HTML = Object.values(LOREM).join('');
export const LOREM_HTML_SPLIT = Object.values(LOREM).join('<hr />');
