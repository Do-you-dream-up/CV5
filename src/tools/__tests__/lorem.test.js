import { LOREM } from '../lorem';

describe('LOREM', () => {
  it('has the expected properties and values', () => {
    expect(LOREM).toHaveProperty('links');
    expect(LOREM).toHaveProperty('lists');
    expect(LOREM).toHaveProperty('paragraphs');

    expect(LOREM.links).toMatchSnapshot();
    expect(LOREM.lists).toMatchSnapshot();
    expect(LOREM.paragraphs).toMatchSnapshot();
  });
});
