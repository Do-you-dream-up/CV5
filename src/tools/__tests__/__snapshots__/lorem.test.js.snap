// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LOREM has the expected properties and values 1`] = `
"
    <h2>Links</h2>
    <p>
      Internal link: <a href>click</a>
      <br />
      External link: <a href target=\\"_blank\\">click</a>
    </p>
  "
`;

exports[`LOREM has the expected properties and values 2`] = `
"
    <h2>Lists</h2>
    <ol>
      <li>Quis lectus nulla at volutpat diam ut venenatis!</li>
      <li>Nibh nisl.</li>
    </ol>
    <ul>
      <li>Pretium quam vulputate dignissim suspendisse in est ante in nibh mauris.</li>
      <li>Cursus mattis molestie a, iaculis at erat?</li>
    </ul>
  "
`;

exports[`LOREM has the expected properties and values 3`] = `
"
    <h2>Paragraphs</h2>
    <p>Pellentesque massa placerat duis ultricies lacus sed turpis tinciduntid aliquet risus.</p>
    <p>Dictum at tempor commodo.</p>
    <p>Ullamcorper a lacus vestibulum sed? Ullamcorper eget nulla facilisi etiam!</p>
  "
`;
